src/Footer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames/bind';
+
+const Footer = ({
+  noComlpetedTodo,
+  onFilteredTodos,
+  clearCompleted,
+  visibleClearCompleted,
+  filterType,
+}) => (
+  <footer className="footer">
+    <span className="todo-count">
+      {noComlpetedTodo}
+      {' '}
+      items left
+    </span>
+
+    <ul className="filters">
+      <li>
+        <a
+          href="#/"
+          className={cn({ selected: filterType === 'All' })}
+          onClick={e => onFilteredTodos(e.target.text)}
+        >
+          All
+        </a>
+      </li>
+
+      <li>
+        <a
+          href="#/active"
+          className={cn({ selected: filterType === 'Active' })}
+          onClick={e => onFilteredTodos(e.target.text)}
```

Comment:
would be great to create an array with data for that buttons and map it to render each button